pkg/verifier/aws/aws_verifier_test.go

```diff
@@ -71,7 +72,7 @@ func TestFindUnreachableEndpointsNoSuccessWithLegacyProbe(t *testing.T) {
 	cli.AwsClient.SetClient(FakeEC2Cli)
 	cli.Logger = &ocmlog.GlogLogger{}
 
-	err := cli.findUnreachableEndpoints(context.TODO(), "dummy-instance", legacy.Probe{})
+	err := cli.findUnreachableEndpoints(context.TODO(), "dummy-instance", legacy.Probe{}, false)
```

Comment:
Can you also add a test for `ensurePrive = true`?

Reply:
can do, let me know if the implementation itself is good. If everything is good, I can add tests accordingly.

Reply:
implementation looks good to me